Dockerfile

```diff
@@ -8,15 +8,18 @@ RUN apk add --update \
   mysql-client \
   busybox-extras \
   ruby python python-dev py-pip \
+  && pip install --upgrade pip \
   && pip install awscli==$AWSCLI_VERSION --upgrade \
   && apk --purge -v del py-pip \
   && rm -rf /var/cache/apk/* \
   && rm -rf $HOME/.cache
 
+RUN go get -u github.com/raelga/liche
```

Comment:
It shouldn't be pointing to personal GH

Reply:
Now points to @dmitry-at-hyla fork, the one that created the PR in the original repo on May. The fix has been merged in the master branch of his fork.